docs/source/reference/visualization.rst

```diff
@@ -1,6 +1,8 @@
 optuna.visualization
 ====================
 
+The :mod:`~optuna.visualization` module provides utility functions for plotting the optimization process using plotly. Plotting functions take generally take a :class:`~optuna.study.Study` object and optional parameters passed as a list to a ``params`` argument.
```

Comment:
Actually, it will be supported by both plotly and matplotlib.
https://github.com/optuna/optuna/pull/1756